src/Tools/dotnet-monitor/DiagnosticsMonitorCommandHandler.cs

```diff
@@ -95,6 +146,7 @@ public IHostBuilder CreateHostBuilder(IConsole console, string[] urls, string[]
                 {
                     // Always allow the experimental tool message to be logged
                     ExperimentalToolLogger.AddLogFilter(builder);
+                    builder.AddFilter(typeof(ApiKeyAuthenticationHandler).FullName, LogLevel.Information);
```

Comment:
Can test the ability to change the log level for this category through configuration? For example, set Microsoft.Diagnostics.Tools.Monitor.ApiKeyAuthenticationHandler to Debug in configuration and make sure that appropriate Debug level message appear.